apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -99,7 +110,7 @@ async function checkRender(
 
   // Errors should generally have stack traces. If they don't, we'll filter
   // them out, but they may not help us much.
-  return ((locals as any).issues as Issue[])
+  return (locals.issues as Issue[])
```

Comment:
Presumably we can drop the `as Issue[]` if this is now correctly typed?

```suggestion
  return (locals.issues)
```

Reply:
Presumably I think #12698 will catch any unneeded type assertions?